engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -354,8 +355,10 @@ class WorkflowManagerActor(backend: Backend, config: Config)
 
     case class ActorAndStateData(actor: WorkflowActorRef, data: WorkflowManagerData)
 
+    // TODO: Obviously unpleasant, but when workflows don't have backends any more, this goes away. Hooray!
```

Comment:
I don't know how everybody else thinks but I'm not a big fan of keeping comments like this in the code.